pkg/suggestion/v1alpha2/test.py

```diff
@@ -0,0 +1,54 @@
+#from . import parsing_utils
```

Comment:
Could we name it random_service_test?

Reply:
I will remove the file for now.  It is not complete

Reply:
We can add tests separately

Reply:
SGTM